php/libraries/NDB_Config.class.inc

```diff
@@ -72,10 +72,23 @@ class NDB_Config
                 // in PHP < 5.5, need to assign to a variable first
                 $configFile = $config->configFilePath();
             }
-
             // Load the file
-            $config->load($configFile);
+            //  load config in test
+            $env    = getenv('LORIS_DB_CONFIG');
+            $travis = getenv('TRAVIS');
+            if (!empty($env)) {
+                if (!empty($travis)) {
+                      $file  = dirname(__DIR__);
+                      $file .= $env;
+                } else {
+                    $file = $env;
```

Comment:
Instead of adding a hack into the code here for Travis, why doesn't the environment just get set properly/to the absolute path in the .travis.yml?

Reply:
This is also one option for docker setup.

Reply:
The absolute path in the .travis.yml depends on user's account.
/home/travis/build/### **kongtiaowang**/Loris/php/home/travis/build/aces/Loris/test/config.xml

Reply:
@driusan The absolute path depends on user's account.
/home/travis/build/**kongtiaowang**/Loris/php
/home/travis/build/**aces**/Loris/php

Reply:
`$TRAVIS_BUILD_DIR/php`?